<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-K322O225" failures="0" tests="8" name="com.salesinventory.SalesInvenTest" time="103.970" errors="1" timestamp="2025-04-03T15:15:57 IST" skipped="0">
  <testcase classname="com.salesinventory.SalesInvenTest" name="createSupplierTest" time="10.940"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="orderTest" time="17.572"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="createEmployeeTest" time="22.820"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="editCustomerButtonTest" time="11.034"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="newCustomerTest" time="13.019"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="newProductTest" time="10.742"/>
  <testcase classname="com.salesinventory.SalesInvenTest" name="transactionValidateTest" time="13.131">
    <error message="Cannot invoke &quot;org.apache.poi.ss.usermodel.Cell.getStringCellValue()&quot; because the return value of &quot;org.apache.poi.ss.usermodel.Row.getCell(int)&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Cell.getStringCellValue()" because the return value of "org.apache.poi.ss.usermodel.Row.getCell(int)" is null
at com.salesinventory.generic.fileutility.ExcelUtility.getDataFromExcel(ExcelUtility.java:17)
at com.salesinventory.SalesInvenTest.transactionValidateTest(SalesInvenTest.java:313)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- transactionValidateTest -->
  <testcase classname="com.salesinventory.SalesInvenTest" name="productCreateTest" time="4.712"/>
</testsuite> <!-- com.salesinventory.SalesInvenTest -->
